app/assets/scripts/components/documents/document-download-menu.js

```diff
@@ -37,6 +40,67 @@ export default function DocumentDownloadMenu(props) {
   const ability = useContextualAbility();
   const canDownloadJournalPdf = ability.can('download-journal-pdf', atbd);
 
+  const handlePdfDownloadClick = React.useCallback(() => {
+    const { id, version, alias } = atbd;
+    const pdfUrl = `${apiUrl}/atbds/${id}/versions/${version}/pdf`;
+    const pdfFileName = `${alias}-v${version}.pdf`;
+    let retryCount = 0;
+    let timeoutRef;
+
+    async function fetchPdf(url) {
+      const toast = createProcessToast('Downloading PDF, please wait...');
+      const user = await Auth.currentAuthenticatedUser();
+      if (!user) {
+        toast.error('Failed to download PDF! (Not authenticated)');
+        return;
+      }
+
+      const {
+        signInUserSession: { idToken, accessToken }
+      } = user;
+      const authToken = `Bearer ${idToken.jwtToken}`;
+      const xAccessToken = accessToken.jwtToken;
+
+      try {
+        const response = await fetch(url, {
+          method: 'GET',
+          headers: {
+            Authorization: authToken,
+            'X-ACCESS-TOKEN': xAccessToken
+          }
+        });
+
+        if (response.headers['content-type'] === 'application/json') {
+          if (retryCount < 3) {
+            window.clearTimeout(timeoutRef);
+            setTimeout(() => {
+              fetchPdf(pdfUrl);
```

Comment:
```suggestion
              fetchPdf(pdfUrl + `?retry=true`);
```
Need to specify `retry=true` for subsequent requests

Reply:
May be we should also check the message body here. When PDF generation is successfully triggered the message looks like this: `{"message":"PDF generation in progress"}`
May be, it would also be good to show a `info` or `success` toast or something similar to let the user know that PDF generation has been successfully triggered - just to make sure that they don't have to wait without any feedback from the UI